src/components/XchangeCourseTable.js

```diff
@@ -47,9 +74,36 @@ function XchangeCourseTable(props) {
             </TableCell>
           </TableRow>
         </TableHead>
-        <TableBody>{props.tableBody}</TableBody>
+        <TableBody>{courseEquivalencies.map((ce) => {
+                  if(ce.uwcourse.id === props.courseId && ce.university.id === props.uniId)
```

Comment:
Just a comment, here we aren't leveraging an endpoint that performs the filtering with an SQL query based on the uni id and are filtering out results in js. 

If there was an endpoint created for this you wouldn't need to conditionally render the results and the view wouldn't be sent irrelevant results.